### DISCUSSION TITLE
black screen

### DISCUSSION THREAD

**Discussion by huoyan1231** [ppy/osu#22460] [Q&A]
### Type

Cosmetic

### Bug description

tab out and tab in

### Screenshots or videos

![image](https://user-images.githubusercontent.com/53113734/215328665-6a67dc25-7261-4a4f-b6be-e4a8d68beee3.png)

### Version

2023.123.0

### Logs

[performance.log](https://github.com/ppy/osu/files/10529825/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/10529826/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/10529827/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/10529828/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/10529829/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10529830/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10529831/updater.log)
[database.log](https://github.com/ppy/osu/files/10529832/database.log)
[input.log](https://github.com/ppy/osu/files/10529833/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10529834/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10529835/network.log)

**Comment by bdach** [comment]
Please try disabling shadowplay and updating graphics drivers.
